### ISSUE TITLE
Allow to change the font size of the tab titles.

### ISSUE THREAD

**Issue by KirilOkun** [manusa/electronim#242]
Got new 32" monitors with 3840x2160 resolution.  It's great for the extra space but the ElectronIM tab titles look very small.  Would be very helpful to be able to change to font size.
Thanks.

**Comment by manusa** [comment]
Hi,
Could you please attach a screenshot. I'm not sure if it's just the font that needs to be addressed, or if this also affects the tab container size.
Also, is the font size also a problem with the settings, about, and help pages?

**Comment by KirilOkun** [comment]
Here's the screen shot.
![Screenshot_20221217_143924](https://user-images.githubusercontent.com/1024870/208231388-f4f4c8a1-eac6-45a5-bda3-7913c618e201.png)

**Comment by KirilOkun** [comment]
Yes,  the settings are a bit on the small size as well.  But since it is used infrequently not a major issue.
